apps/client/src/routes/inn.index.lazy.tsx

```
@@ -0,0 +1,28 @@
+      .then((response) => response.text())
+      .then((html) => setHtmlContent(html))
+      .catch((error) => console.error("Failed to fetch HTML:", error));
+  }, []);
+
+  if (!htmlContent) {
+    return <p>Loading...</p>;
+  }
+
+  return (
+    <div
+      dangerouslySetInnerHTML={{ __html: htmlContent }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)